app.py

```diff
@@ -121,25 +113,7 @@
 
 routes = [
     Route("/full_config", FullConfigRest),
-    Route("/notify/{path:path}", Notify),
-    Route("/healthcheck", Healthcheck),
```

Comment:
i think these two are still needed 
Notify is the minio endpoint, called from a different container (e.g. for our project file upload)
Heathcheck is called by the better stack.

Refinery project upload also doesn't work atm in branch

Reply:
Nicely spotted, adding those back and retesting with import.